openshift/log-detective.yaml

```diff
@@ -2,23 +2,23 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: log-detective-website
+  name: logdetective-website
 spec:
   selector:
     matchLabels:
-      component: log-detective-website
+      component: logdetective-website
   template:
     metadata:
       labels:
-        component: log-detective-website
+        component: logdetective-website
     spec:
       volumes:
         - name: persistent
```

Comment:
thank god this doesn't need changing, we'd have to migrate the data

Reply:
I'm going to remove this. Although it would be better to make sure names match everywhere.